### DISCUSSION TITLE
VICTORPB - UNDISCORD HAS NOT WORKED IN MONTHS

### DISCUSSION THREAD

**Discussion by gaveitatry2** [victornpb/undiscord#511] [4. Bugs and Issues]
@victornpb

https://github.com/victornpb/undiscord/issues/509

https://github.com/victornpb/undiscord/issues/485

https://github.com/victornpb/undiscord/issues/487

https://github.com/victornpb/undiscord/issues/491

https://github.com/victornpb/undiscord/issues/492

https://github.com/victornpb/undiscord/issues/495

https://github.com/victornpb/undiscord/issues/496

https://github.com/victornpb/undiscord/issues/499

https://github.com/victornpb/undiscord/issues/507

https://github.com/victornpb/undiscord/issues/509

Please fix your Undiscord userscript. Everyone has been experiencing 401 and 403 errors for the last 2.5 months. Undiscord won't delete ANY MESSAGES 99% of the time. There have been DOZENS of threads in the last TWO AND A HALF MONTHS with screenshots and complaints. Please do something about it. We have tried the older Undiscord versions too, but they don't even work (99% of the time).

**Comment by gaveitatry2** [comment]
Update: It sometimes works, but a LOT OF THE TIME it doesn't. It's sporadic. Someone mentioned this in one of the issues I linked to. How sometimes it works for them and sometimes doesn't.

**Comment by gaveitatry2** [comment]
Update: The few times it works, it doesn't even work right. I had 33 messages that needed deleted. It deleted 25 and then stopped working and started throwing error messages.

I don't know what the guy above me is talking about. I was already using authorization tokens. And none of my messages were triggered by bot slash commands. It sounds like he desperately wants to prove this userscript is working when it isn't, hence the dozen Issues created in the last 2.5 months by other people who aren't me.

**Comment by 573rn** [comment]


**Reply by gaveitatry2** [reply]
Nice try, but I'm not using whitespace in posts and not trying to delete posts that have the word "warn". Sometimes I can have 200 messages in a server and this Undiscord wont' delete a single one of the messages. Why don't you stop wasting everyone's time with BS advice like this?

**Reply by allgamezzz** [reply]
I second all your points in this post and echo them @gaveitatry2
Thanks for saying something.

**Comment by CountingCard** [comment]
Not sure if this is the case here, but I've noticed that some browsers will actually autofill your username on some of the settings in the script. For example, under "Message Interval" it autofilled my Email, and in the "Authorization Token", it autofilled my password. Go through and make sure those prompts are correct. Let me know if this solves your issue or if my browser was just being dumb. The script currently works for me after those fixes.
